packages/api-server/src/util.ts

```diff
@@ -41,7 +41,7 @@ export function snakeToCamel(
   let camel: any = {};
   Object.entries(t).map(([key, value]) => {
     let newValue =
-      typeof value === "object"
+      typeof value === "object" && value != null
```

Comment:
In my opinion, the better idea might be to handle type detection at the beginning of snakeToCamel, just [as the npm package "snake-camel" does](https://github.com/craftgear/snake-camel/blob/master/src/index.ts#L20-L22):

```ts
function camelToSnake(t, ..) {
  IF t is not type object {
    RETURN t;
  }

  ...
}
```

Reply:
updated